app-arch/lzfse/lzfse-1.0.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+CMAKE_INSTALL_PREFIX=""
+
+inherit cmake
+
+DESCRIPTION="LZFSE compression utilities"
+HOMEPAGE="https://github.com/lzfse/lzfse"
+SRC_URI="https://github.com/lzfse/lzfse/archive/refs/tags/lzfse-${PV}.tar.gz"
+S="${WORKDIR}/${PN}-lzfse-${PV}"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="arm64"
```

Comment:
I'd prefer it if we didn't commit it as stable. It's OK if you do it in a followup commit if you really need to have it stable (we may have discussed that before, I don't recall) but do it in a separate one if so, and then the commit message can explain it rather than it looking accidental.

Reply:
This was just a mistake. Committed the package as copied from the overlay without changing to `~arm64`. It will need to be added as stable so that releng can build working stable livecds, but I will do it in a separate commit like with everything else so far.

Reply:
Wait no, I did do it correctly?

Reply:
You did! Sorry!